full/src/main/java/apoc/custom/CypherProceduresHandler.java

```diff
@@ -116,13 +120,16 @@ public CypherProceduresHandler(GraphDatabaseAPI db, JobScheduler jobScheduler, A
 
     @Override
     public void available() {
+        // I restore procs and function even with apoc.custom.procedures.enabled=false to not create inconsistency with system nodes
```

Comment:
questa non l'ho capita

Reply:
Perché in pratica potresti creare alcune procedure (con enabled=true), e quindi gli associati nodi di sistema.
Poi se si disabilitano le custom procedure e si riavvia, se non richiamo `restoreProceduresAndFunctions()` potrei avere la `apoc.custom.list()` vuota mentre i nodi di sistema rimarrebbero. Ho aggiornato il commento.